src/app/patient-dashboard/common/patient-info/patient-demographics.component.html

```diff
@@ -22,7 +22,7 @@
        </tr>
          <tr>
            <td><strong>Age  : </strong> {{patient.person.age}}</td>
-         <td>&nbsp;</td>
+           <td><strong>Highest Educations  : </strong>{{patientLevelOfEducation?.name.display}}</td>
```

Comment:
Maybe ought to be `Highest Education Level`?

Reply:
:+1: